hiss/templates/dashboard_base.html

```diff
@@ -13,12 +13,14 @@
                         </button>
                         <div class="collapse navbar-collapse" id="navbarSupportedContent">
                             <a class="nav-item" href="{% url 'status' %}">Status</span></a>
-                            {% if not user.team %}
-                                <a class="nav-item" href="{% url 'team:create'  %}">Create Team</a>
-                                <a class="nav-item" href="{% url 'team:join'  %}">Join Team</a>
-                            {% else %}
-                                <a class="nav-item" href="{{ user.team.get_absolute_url }}">My Team</a>
-                            {% endif %}                                                 
+                            {% if not NEEDS_TO_APPLY %}
```

Comment:
This variable is not always provided to `dashboard_base.html`. Other views (like creating an application, updating an application, etc.) don't provide this context, and so this would malfunction in those cases.

For example, if a user is creating an application, since `NEEDS_TO_APPLY` is not provided, this statement evaluates to true, and `My Team` would be shown.

Please use the global `user` object instead.

```suggestion
                            {% if not NEEDS_TO_APPLY and user.application %}
```